### DISCUSSION TITLE
avatar - no info placeholder

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#954] [Office hours] [answered]
What should we use if there is no name and no image for the avatar?

In my contact center use case, we want to be able to show an avatar of the participant if the client has and wants that information shown.

There may be a case where the image or name is missing. What should be shown there instead?

I've found a "user" icon and made this as a suggestion
![image](https://user-images.githubusercontent.com/6718764/99566546-e8e87300-2981-11eb-847d-6e3f42b928d8.png)

Alternatively, if we can't use something like this or if there isn't a solution, what about replacing the name initials with text like a `?`  or some other suggestion?

**Comment by kate-porter** [comment]
For now, Elaine and team are going to go with the `?` for an empty state, until we create an empty state for Anchor.

**Comment by nkrantz** [comment, marked as answer]
Following up on this thread - the Avatar component now has an `icon` prop which you can use to display any Paste Icon within an Avatar. [Read more on the documentation site.](https://paste.twilio.design/components/avatar#representing-an-entity-via-an-icon)
